src/database/SsiCredentialIssuer.Migrations/Seeder/Data/use_cases.json

```diff
@@ -43,5 +43,10 @@
     "id": "3793a2d9-6bc5-44f0-b952-7d3f6b747dd7",
     "name": "Demand and Capacity Management",
     "shortname": "Puris"
+  },
+  {
+    "id": "1aacde78-35ec-4df3-ba1e-f988cddcbbd9",
```

Comment:
what is this dataset good for? It's not linked anywhere

Reply:
linkage to use case none was added.